presto-memory/src/main/java/com/facebook/presto/plugin/memory/MemoryMetadata.java

```diff
@@ -72,6 +75,7 @@
     private final Map<SchemaTableName, Long> tableIds = new HashMap<>();
     private final Map<Long, MemoryTableHandle> tables = new HashMap<>();
     private final Map<Long, Map<HostAddress, MemoryDataFragment>> tableDataFragments = new HashMap<>();
+    private final Map<SchemaTableName, String> views = new HashMap<>();
```

Comment:
This should actually have `@GuardedBy("this")` (explained in the *Java Concurrency in Practice* book), but none of the others do, so we'll leave it out for now and make that change separately.